ci/travis-after-success.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -xe
 
+# Submit code coverage report
+curl -s https://codecov.io/bash | bash
```

Comment:
Oh, this is nasty. I don't like it at all, is there anything that doesn't involve executing a remote script to upload the coverage report?